src/tools/analyzer.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+permalink: /tools/analyzer
+title: "Static Analyzer"
+description: "The tool that analyzes and validates your Dart code."
+---
+
+The static analyzer evaluates your Dart code,
+checking for errors and warnings that are specified in the
+[Dart Language Specification](https://www.dartlang.org/docs/spec/).
+
+Many tools in the Dart ecosystem use the [analysis
+server](https://github.com/dart-lang/sdk/tree/master/pkg/analysis_server)
+to perform static analysis. For example:
+
+* Dart and Flutter plugins for IDEs (such as
+  [WebStorm](https://webdev.dartlang.org/tools/webstorm),
+  [IntelliJ](https://www.dartlang.org/tools/jetbrains-plugin), and
+  [Atom](https://atom.io/packages/dartlang))
+* [DartPad](https://www.dartlang.org/tools/dartpad)&mdash;browser-based
+  Dart playground
+* [Dart Dev Compiler (DDC)](https://github.com/dart-lang/sdk/tree/master/pkg/dev_compiler)&mdash;experimental development tool and transpiler for web apps
+* [`dartanalyzer`](https://github.com/dart-lang/sdk/tree/master/pkg/analyzer_cli#dartanalyzer
+)&mdash;command-line static analyzer for web or VM apps
+* [`flutter analyze`](https://flutter.io/debugging/#the-dart-analyzer)&mdash;command-line
+  static analyzer for mobile (Flutter) apps
+
+## Customizing static analysis
+
+You can customize the behavior of the analyzer or the linter
+by adding an analysis options file (`analysis_options.yaml`) to
+the package root of your project. The linter, an analyzer plugin,
+allows you to define (or exclude) rules specific to your project.
+For example, you can add the `close_sinks` rule to ensure that all
+sink methods include a `close()` call. For further information, see
+[Customize Static Analysis](/guides/language/analysis-options).
+
+## Building tools that use static analysis
```

Comment:
I think that there are two distinct groups of readers: developers that are using Dart to build a web or mobile app and developers that are building development tools for Dart. I think this section ("Building tools that use static analysis") is targeted toward the second, but I think this document should be targeted toward the first. Hence, I think this section should be moved to some other document. (I think it will be confusing for users that aren't trying to build tooling, despite the title.)